web/scss/_roadmap.scss

```diff
@@ -118,7 +118,7 @@ $title-spur: 8 * $width-roadmap;
   font-size: 0.9em;
   margin: 10px;
 }
-
+.roadmap-item-content a{color:$color-primary;}
```

Comment:
I think the proper solution to this is just to apply the `primary-link` tag to the anchor.

Even if a custom rule was preferred here, then it should presumably still use `@include anchor($color-primary)` as is done elsewhere.

Reply:
It'd also probably be acceptable to wrap the roadmap description items in the CSS class `markdown`.